src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -152,59 +171,67 @@ const StudentApplication = ({ userProfile, authentication }) => {
   const isApplicantValid = async (applicant) => {
     // Check that the applicant contains the required fields
     if (applicant?.Profile === null) {
+      createSnackbar(
+        'Something went wrong while trying to add this person. Please try again.',
+        'error',
+      );
       return false;
     }
 
-    if (applicant.Profile.fullName === null) {
-      applicant.Profile = user.formatName(applicant.Profile);
-    }
-
-    if (applicant?.OffCampusProgram === null) {
-      applicant.OffCampusProgram = '';
-    }
-
     if (applicationDetails.Applicants.length >= MAX_NUM_APPLICANTS) {
       // Display an error if the user try to add an applicant when the list is full
-      setSnackbarText(`You cannot have more than ${MAX_NUM_APPLICANTS} applicants'`);
-      setSnackbarSeverity('warning');
+      createSnackbar(`You cannot have more than ${MAX_NUM_APPLICANTS} applicants`, 'warning');
       return false;
     }
 
+    applicant.Profile.fullName ??
+      (applicant.Profile.fullName = `${applicant.Profile.FirstName}  ${applicant.Profile.LastName}`);
```

Comment:
This would be another opportunity for the nullish assignment operator:
```suggestion
    applicant.Profile.fullName ??= `${applicant.Profile.FirstName}  ${applicant.Profile.LastName}`;
```